cmd/kube-apiserver/app/options/options.go

```diff
@@ -58,6 +58,7 @@ type ServerRunOptions struct {
 	EgressSelector          *genericoptions.EgressSelectorOptions
 	Metrics                 *metrics.Options
 	Logs                    *logs.Options
+	OpenTelemetry           *genericoptions.OpenTelemetryOptions
```

Comment:
we should probably eventually refactor this into flags defined from `component-base/traces` (but not necessarily right now)

Reply:
is this kube-apiserver specific or would it be applicable to any apiserver?

Reply:
I would expect this in every apiserver. Though this does't make it misplaced. But it should be part of `RecommendedOptions` too if it is't yet.

Reply:
I added it to RecommendedOptions.